File: CS/CustomLocalizedString/MyStringLoader.cs
using DevExpress.Maui.Core;

namespace CustomLocalizedString;

public class MyStringLoader : Localizer.IStringLoader
{
    public bool TryGetString(string key, out string value)
    {
        if (key == "SchedulerStringId.AppointmentEdit_NewAppointmentTitle")
        {
            value = "Nueva Cita";
            return true;
        }

        value = null;
        return false;
    }
}

File: VB/CustomLocalizedString/MyStringLoader.vb
Imports DevExpress.Maui.Core
Imports System.Runtime.InteropServices

Namespace CustomLocalizedString

    Public Class MyStringLoader
        Inherits Localizer.IStringLoader

        Public Function TryGetString(ByVal key As String, <Out> ByRef value As String) As Boolean
            If Equals(key, "SchedulerStringId.AppointmentEdit_NewAppointmentTitle") Then
                value = "Nueva Cita"
                Return True
            End If

            value = Nothing
            Return False
        End Function
    End Class
End Namespace
